path_rotate.go

```diff
@@ -7,14 +7,20 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"math"
 	"time"
 
 	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/helper/backoff"
 	"github.com/hashicorp/vault/sdk/logical"
 	"github.com/hashicorp/vault/sdk/queue"
 )
 
+var (
+	rollbackAttempts    = 10
+	minRollbackDuration = 1 * time.Second
+	maxRollbackDuration = 100 * time.Second
```

Comment:
curious how we decided on the value for what a good max duration would be

Reply:
I think those are based on the previous backoff implementation.

Reply:
Yep, I didn't want to go significantly higher than the previous max, which was 81 seconds.